test/suite/hook_tiered.py

```diff
@@ -124,17 +217,43 @@ def filter_tests(self, tests):
         return new_tests
 
     def setup_hooks(self):
+        orig_session_alter = self.Session['alter']
+        self.Session['alter'] =  (wthooks.HOOK_REPLACE, lambda s, uri, config:
+          session_alter_replace(orig_session_alter, s, uri, config))
+
+        orig_session_checkpoint = self.Session['checkpoint']
+        self.Session['checkpoint'] = (wthooks.HOOK_REPLACE, lambda s, config=None:
+          session_checkpoint_replace(orig_session_checkpoint, s, config))
+
+        orig_session_close = self.Session['close']
+        self.Session['close'] = (wthooks.HOOK_REPLACE, lambda s, config=None:
+          session_close_replace(orig_session_close, s, config))
```

Comment:
Hmm, almost no tests use session.close itself. Few use connection.close other than just falling off the end of the test and the system infrastructure calls connection.close.
@ddanderson do the python internals call session.close itself?

Reply:
@sueloverso, you're right.  Very few tests explicitly call `session.close()`.  When a test completes, the testing framework closes any connections that were opened, but that won't cause any `session.close` API calls that you can hook.

Reply:
Can the hooks grab/replace connection.close instead of session.close? Or are session methods the only thing available?

Reply:
OK.  I'll just remove this hook then rather than leaving it there and implying that it gets called with any frequency.